forest/benchmarking/tests/test_utils.py

```diff
@@ -64,17 +64,17 @@ def test_bitstring_prep():
                                             'RX(0) 5']
     # mixed flips
     flip_prog = bitstring_prep([0, 1, 2, 3, 4, 5], [1, 1, 0, 1, 0, 1])
-    assert flip_prog.out().splitlines() == ['RX(pi) 0',
-                                            'RX(pi) 1',
+    assert flip_prog.out().splitlines() == [f'RX({np.pi}) 0',
+                                            f'RX({np.pi}) 1',
```

Comment:
These occurrences of `np.pi` in the program output (`.out()`) suggest a breach of the Quil spec to me:
https://github.com/quil-lang/quil/blob/master/grammars/Quil.g4

The grammar says it should be represented as `pi`.

Reply:
The `np.pi` is interpolated in the `f` string, so the output comparison actually looks like 
```py
assert flip_prog.out().splitlines() == ['RX(3.141592653589793) 0',
                                        'RX(3.141592653589793) 1',
```

Part of upgrading to `4.0` was changing the parser to use `quil-rs`, which since https://github.com/rigetti/quil-rs/pull/240 will convert the `pi` literal into the float value.

Reply:
Ah, yep I misread this in review. All good.